### ISSUE TITLE
Windoor dir density issues

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#30952]
(WEB REPORT BY: kanef REMOTE: 206.221.180.138:7777)
# Revision
2d2320d3e60f0f1d7a53470672047a75bf607a5c

# Description
Windoors have issues with directional density and interactions such as machine usage or item grabbing
# Steps to Reproduce
* Place windoor
* Grab item in its tile or away from it, same with machine usage
# What you Expected
To use it if not blocking it
# What Actually Happened
Tile blocked wholesale

**Comment by ghost** [comment]
Update: should mention the dir on the windoor was 6, a diagonal one, if that helps.

**Comment by Exxion** [comment]
Yeah that would definitely be why it happened, but how did it get a diagonal direction?

**Comment by ghost** [comment]
> Yeah that would definitely be why it happened, but how did it get a diagonal direction?

Not even sure, just appeared there in the round somehow. Maybe it got dpushed by wiz? I'll have to logdive

**Comment by GreatBigFailure** [comment]
One weird interaction here is that windows can contain infinite pressure but only sometimes. For example, the boxing ring north of the bar on box will explode into glass shrapnel if you throw some molotavs inside.

**Comment by Exxion** [comment]
Yes, windows break if there is high pressure on their own tile, but do not care about the tile they're facing. That's not related to this issue, though.

EDIT: Ah! I fell victim to the classic myth!
Windows do not care about pressure at all. They are damaged by *fire* on their own tile, but not the one they're facing.

**Comment by Zth--** [comment]
Huh, this may be related to runechat sometimes going _under_ some windoors, which I find super annoying
